TransformationSystem/Agent/TaskManagerAgentBase.py

```diff
@@ -254,12 +247,28 @@ def _getClients(self):
     return {'TransformationClient': threadTransformationClient,
             'TaskManager': threadTaskManager}
 
+  def _getDelegatedClients(self, ownerDN=None, ownerGroup=None):
```

Comment:
Can you simply modify `_getClients()` instead of adding one more function? For extensions is one less thing to extend.

Reply:
yes, moved the parameters into _getClients